src/ray/core_worker/lib/java/io_ray_runtime_gcs_GlobalStateAccessor.cc

```diff
@@ -28,8 +29,12 @@ Java_io_ray_runtime_gcs_GlobalStateAccessor_nativeCreateGlobalStateAccessor(
     JNIEnv *env, jobject o, jstring j_redis_address, jstring j_redis_passowrd) {
   std::string redis_address = JavaStringToNativeString(env, j_redis_address);
   std::string redis_password = JavaStringToNativeString(env, j_redis_passowrd);
-  gcs::GlobalStateAccessor *gcs_accessor =
-      new gcs::GlobalStateAccessor(redis_address, redis_password);
+  std::vector<std::string> results;
```

Comment:
Maybe parsing redis address to create GcsClientOptions can be a common helper function?

Reply:
No sure we need that or not. It's only in java code right now. In python, it has a converter there.

Reply:
I just think the logic is duplicated in `ProcessHelper::CreateGlobalStateAccessor()` too. But the logic is short so keeping them duplicated is fine.